DotNet/DataAcquisition.Domain/Application/Services/PatientDataService.cs

```diff
@@ -255,7 +255,7 @@ await _dataAcquisitionLogManager.CreateAsync(
                     }
                     catch (Exception ex)
                     {
-                        var message = "Error creating log entry for facility {facilityId} and patient {patientId}\n{ex.Message}\n{innerException}";
+                        var message = "Error creating log entry for facility {facilityId} and patient {patientId}";
```

Comment:
This might cause issues with CA2554 when #1168 is merged getting merged.

Reply:
Resolved in `44901635` by moving the template `message` into the `LogError` call (as a string literal).